src/SBH_LinearRegressionChart.js

```diff
@@ -145,6 +145,12 @@ class SBH_LinearRegressionChart extends React.Component {
   }
 }
 
-SBH_LinearRegressionChart.propTypes = {};
+SBH_LinearRegressionChart.propTypes = {
+  title: PropTypes.string,
+  scatterData: PropTypes.object,
+  lineData: PropTypes.object,
+  xAxisDataKey: PropTypes.string,
+  yAxisDataKey: PropTypes.string,
```

Comment:
If a prop is a must be provided, you have to add 'isRequired' to prop type.

For example,
`lineData: PropTypes.object.isRequired`

Please add 'isRequired' to the props that must be provided.
I think all of the props except title must be provided.